woocommerce/woocommerce-ios
# Jetpack CP: Update Networking and Storage with adminURL

Part of #5365 


### Description

This PR updates Site models in Networking and Storage with new property / attribute `adminURL`, which will be used to navigate user to WPAdmin page.

To get `adminURL` in the site list response, a new param `admin_url` is added to `options` list in the `/me/sites` request.

The 700+ line change is due to the new model version.

### Testing instructions

Build the app and make sure that there's no crash. Site list should be loaded successfully as before.

### Screenshots

N/A


---
